lib/aliases.zsh

```diff
@@ -1,35 +0,0 @@
-# Push and pop directories on directory stack
-alias pu='pushd'
-alias po='popd'
-
-# Basic directory operations
-alias ...='cd ../..'
-alias -- -='cd -'
```

Comment:
It's the same as `1`: https://github.com/robbyrussell/oh-my-zsh/pull/3564/files#diff-f7d0a3ef1bc4893bc37a320a5c593e70L13

You can add the alias at the end of your zshrc file, or if more people want it back, we can add it again to `directories.zsh`